modules/aggregation-layers/src/contour-layer/value-reader.ts

```diff
@@ -0,0 +1,63 @@
+import {Aggregator} from '../aggregation-layer-v9/aggregator';
+import {WebGLAggregator} from '../aggregation-layer-v9/gpu-aggregator/webgl-aggregator';
+import {CPUAggregator} from '../aggregation-layer-v9/cpu-aggregator/cpu-aggregator';
+import type {TypedArray} from '@luma.gl/core';
+
+type ValueReader = (x: number, y: number) => number;
+
+/** Returns an accessor to the aggregated values from bin id */
+export function getAggregatorValueReader(opts: {
+  aggregator: Aggregator;
+  binIdRange: [number, number][];
+  channel: 0 | 1 | 2;
+}): ValueReader | null {
+  const {aggregator, binIdRange, channel} = opts;
+
+  if (aggregator instanceof WebGLAggregator) {
+    const buffer = aggregator.getResult(channel)?.buffer;
+    if (buffer) {
```

Comment:
Nit: Is there really no way of adding a `getValueReader()` type method to the aggregator interface so that we don't need this abstraction-breaking instanceof fiddling in the layers?

Reply:
I'm on the fence about this. This functionality is only really needed by this layer.